### DISCUSSION TITLE
p/d0d2e2c3e6364759aaddb419691f937e/

### DISCUSSION THREAD

**Discussion by giscus** [rxliuli/blog#22] [Announcements]
# p/d0d2e2c3e6364759aaddb419691f937e/

周六早晨，吾辈收到了一封来自 openai 的邮件，说支付失败，如果在 04-22 号之前没有处理，账号将会被禁用。  吾辈之前曾使用虚拟信用卡的方式购买过 plus，也支付了两次，本以为付费用户的信用卡不会被禁用（否则怎么付费呢？），结果还是太天真了。再又尝试开了几张卡之后，也尝试了淘宝的高昂手续费，但仍然无法支付成功。颇有一种 openai 站着挣钱，吾辈跪着送钱的感觉，或许国内只是不配用 o

https://blog.rxliuli.com/p/d0d2e2c3e6364759aaddb419691f937e/

**Comment by ilyydy** [comment]
可以用 azure 的 openai， https://learn.microsoft.com/en-us/azure/cognitive-services/openai/

**Reply by rxliuli** [reply]
看了一下，似乎只供企业用户使用

**Reply by ilyydy** [reply]
我申请注册的审核不严，主要是要用非个人邮箱，通过之后可以把账号的主邮箱换成个人邮箱
